### ISSUE TITLE
Little Error in official document

### ISSUE THREAD

**Issue by wikke** [reiinakano/scikit-plot#34]
http://scikit-plot.readthedocs.io/en/stable/apidocs.html#classifier-plots

in plot_confusion_matrix paragraph, the code is 
```
 rf = classifier_factory(RandomForestClassifier())
 rf.plot_learning_curve(X, y, normalize=True)
<matplotlib.axes._subplots.AxesSubplot object at 0x7fe967d64490>
plt.show()
```

it should be
```
 rf = classifier_factory(RandomForestClassifier())
 rf.**plot_confusion_matrix**(X, y, normalize=True)
<matplotlib.axes._subplots.AxesSubplot object at 0x7fe967d64490>
plt.show()
```

**Comment by reiinakano** [comment]
Good catch! If you'd like to submit a PR to fix it yourself, the relevant line is in https://github.com/reiinakano/scikit-plot/blob/master/scikitplot/classifiers.py#L124

If not, then I'll get to it in a day or two. :) Thanks for catching this!

**Comment by wikke** [comment]
Notice your reply late. It's my pleasure to pull request, already pull.

**Comment by reiinakano** [comment]
Fixed in #35
